python/graphstorm/gconstruct/transform.py

```diff
@@ -101,6 +103,84 @@ def __call__(self, strs):
                 atten_mask_name: th.cat(att_masks, dim=0).numpy(),
                 token_type_id_name: th.cat(type_ids, dim=0).numpy()}
 
+class ComputeBERT(FeatTransform):
```

Comment:
Because this class extends from FeatTransform, can we rename is to BERTTransform or similar. ComputeBERT is an action, not sounds like a noun.